browser/src/control/Control.PartsPreview.js

```diff
@@ -185,6 +185,8 @@ L.Control.PartsPreview = L.Control.extend({
 		var img = L.DomUtil.create('img', imgClassName, frame);
 		img.setAttribute('alt', _('preview of page ') + String(i + 1));
 		img.setAttribute('tabindex', '0');
+		img.setAttribute('data-cooltip', _('Slide ') + String(i + 1));
```

Comment:
no spaces at the end of string, also it's not a good approach. In some languages word order might be different.
we really need to use token and replace it later:
`_('Slide %1').replace('%1', String(i + 1));`

Reply:
the same should be done for preview of page too

Reply:
cc: @timar am I correct?